src/routes/+layout.svelte

```diff
@@ -44,3 +37,6 @@
 {:else}
 	<slot />
 {/if}
+<!--
+     afn if you are moving the footer here, please make it not use the repositories store directly and instead use component props :) -->
```

Comment:
:dev_happy_emoji:

Reply:
I love making the lives of other contributors easier